pkg/applyinator/applyinator.go

```diff
@@ -43,22 +76,42 @@ func NewApplyinator(workDir string, preserveWorkDir bool, appliedPlanDir string,
 	}
 }
 
-func (a *Applyinator) Apply(ctx context.Context, anp types.AgentNodePlan) ([]byte, error) {
-	logrus.Infof("Applying plan with checksum %s", anp.Checksum)
+func CalculatePlan(rawPlanByteData []byte) (CalculatedPlan, error) {
```

Comment:
Can you remove the Hungarian notation? It provides an unnecessary redundancy.

Reply:
Hungarian notation removed